db/data_migration/20181112120518_rural_payments_agency.rb

```diff
@@ -0,0 +1,327 @@
+  'hedgerows-and-boundaries-grant-manual-2017-countryside-stewardship',
+  'countryside-stewardship-hedgerows-and-boundaries-grant-manual-2016',
+  'guide-to-countryside-stewardship-facilitation-fund',
+  'countryside-stewardship-woodland-management-plan-grant-manual-2017',
+  'countryside-stewardship-woodland-creation-grant-manual-2018',
+  'high-speed-two-hs2-how-it-affects-agri-environment-scheme-agreements',
+  'countryside-stewardship-woodland-tree-health-grant-manual-2017',
+  'woodland-tree-health-grant-countryside-stewardship',
+  'southern-north-sea-marine-area-index-map-and-site-packages',
+  'hedgerows-and-boundaries-grant-countryside-stewardship',
+  'water-environment-grant-weg-handbooks-guidance-and-forms',
+  'woodland-management-plan-grant-countryside-stewardship',
+  'annex-8-of-mid-tier-and-new-countryside-stewardship-offers-for-wildlife-manual',
+  'countryside-stewardship-how-to-submit-claims-online',
+  'hot-and-dry-weather-conditions-in-england',
+  'historic-buildings-restoration-grant-countryside-stewardship',
+  'catchment-sensitive-farming-officer-contacts',
+  'countryside-stewardship-manual-and-grants-1-january-2016-agreements',
+  'countryside-stewardship-mid-tier-including-water-quality-capital-items-manual-for-1-january-2017-agreements',
+  'countryside-stewardship-higher-tier-manual-for-1-january-2017-agreements',
+  'mid-tier-manual-for-1-january-2018-agreements-countryside-stewardship',
+  'higher-tier-manual-for-1-january-2018-agreements-countryside-stewardship',
+  'countryside-stewardship-higher-tier-manual',
+  'countryside-stewardship-mid-tier-including-water-quality-capital-items-manual',
+  'arable-offer-countryside-stewardship',
+  'lowland-grazing-offer-countryside-stewardship',
+  'mixed-farming-offer-countryside-stewardship',
+  'upland-offer-countryside-stewardship',
+  'countryside-stewardship-facilitation-fund-successful-applications',
+  'countryside-stewardship-facilitation-fund-successful-applications-in-round-2',
+  'countryside-stewardship-facilitation-fund-successful-applications-in-2017-floods-round',
+  'facilitation-fund-successful-applications-in-2017-national-round',
+  'countryside-stewardship-flood-reduction-infographic',
+  'countryside-stewardship-facilitation-fund-case-studies',
+  'mid-tier-events-2018-countryside-stewardship',
+  'tell-the-rural-payments-agency-about-land-changes-and-entitlement-transfers',
+].each do |slug|
+  begin
+    document = Document.find_by(slug: slug)
+    edition = document.latest_edition
+
+    lead_organisations = edition.lead_organisations.to_a
+    supporting_organisations = edition.supporting_organisations.to_a
+
+    # replace `old` with `new` in the lead and supporting orgs; if
+    # `old` is lead and `new` is supporting, then this will "promote"
+    # `new` to lead.
+    if lead_organisations.include? old
+      add_to_list(lead_organisations, new)
+      supporting_organisations.delete new
```

Comment:
You're right, I think the other case should be `elsif supporting_organisations.include?(old) && !lead_organisations.include?(new)`.  Otherwise, in the situation where `new` is in `lead_organisations` and `old` is in `supporting_organisations` we'll end up with `new` in both.